packages/blocks/README.md

```diff
@@ -730,6 +730,7 @@ _Parameters_
 
 -   _blocks_ `(Array|Object)`: Blocks array or block object.
 -   _name_ `string`: Block name.
+-   _isGroupingBlock_ `boolean`: Set to true to bypass default grouping block selection.
```

Comment:
What does this do? Is bock transform normally wrapped by a group block or something?